distribution/docs/src/main/resources/content/introduction.adoc

```diff
@@ -0,0 +1,11 @@
+:type: introduction
```

Comment:
This empty .adoc file is needed to create a new document. All of its contents will be overwritten by the templates, but it will create an html and pdf with this file name.